src/models.py

```diff
@@ -98,8 +98,12 @@ def __init__(self, model, client):
                 if callable(attr):
                     self.should_index = attr
                 else:
-                    raise AlgoliaIndexError('{} should be a callable.'.format(
-                        self.should_index))
+                    should_index_attr_name = self.should_index
+                    if isinstance(attr, bool):
```

Comment:
I added comment on the test cases below: this line `if isinstance(attr, bool)` with lines 106-107 needs to go or be moved somewhere else :)